pkg/config_test.go

```diff
@@ -40,6 +41,18 @@ func TestBadConfigs(t *testing.T) {
 			configFile: "unknown_version.bad.yml",
 			errorMsg:   "apiVersion line missing or version is unknown (invalidVersion)",
 		},
+		{
+			configFile: "custom_dimension_without_name.yml",
+			errorMsg:   "Name should not be empty",
```

Comment:
Nitpick requests:
- test files could be renamed to `custom_namespace_...` now that the config name has changed
- could we keep the pattern of using `.ok.yml` and `.bad.yml` extensions?

Reply:
done!